src/test/java/it/gov/pagopa/payment/dto/mapper/TransactionInProgress2SyncTrxStatusTest.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.payment.dto.mapper;
+
+import org.junit.jupiter.api.Test;
+
+
+
+class TransactionInProgress2SyncTrxStatusTest {
+
+    @Test
+    void transactionInProgressMapper() {
```

Comment:
let's remember to check if there are not null fields (except what should be expected to be null)
it.gov.pagopa.payment.test.utils.TestUtils#checkNotNullFields